ext/soap/php_xml.c

```diff
@@ -98,7 +90,7 @@ xmlDocPtr soap_xmlParseFile(const char *filename)
 		ctxt->sax->warning = NULL;
 		ctxt->sax->error = NULL;
 		/*ctxt->sax->fatalError = NULL;*/
-		old = php_libxml_disable_entity_loader(1);
+		bool old = php_libxml_disable_entity_loader(1);
```

Comment:
Nit: while at it, I took an embarrassing amount of time to not find this weird compared to the line below using `old`.
```suggestion
		bool old = php_libxml_disable_entity_loader(true);
```